terraform/load-balancing.tf

```diff
@@ -67,6 +98,30 @@ resource "aws_lb_listener" "fullnode-ac" {
   }
 }
 
+resource "aws_lb_target_group" "fullnode-jsonrpc" {
+  name     = "${terraform.workspace}-fullnode-jsonrpc"
+  protocol = "TCP"
+  port     = 5000
+  vpc_id   = aws_vpc.testnet.id
+}
+
+resource "aws_lb_target_group_attachment" "fullnode-jsonrpc" {
+  count            = var.num_fullnodes
+  target_group_arn = aws_lb_target_group.fullnode-jsonrpc.arn
+  target_id        = element(aws_instance.fullnode.*.id, count.index)
+}
+
+resource "aws_lb_listener" "fullnode-jsonrpc" {
+  load_balancer_arn = aws_lb.validator-jsonrpc.arn
+  port              = 5001
```

Comment:
Why is the validator and fullnode both being exposed? For AC only the fullnode is.

Reply:
@mgorven So fullnode will still be using AC? only fullnode to validator is through json-rpc?

Reply:
no, AC will be removed from both Validator and FN, but enabling json-rpc endpoint on Validator is unnecessary (clients should connect to FNs instead)